.buildkite/autogenerate_pipeline.py

```diff
@@ -41,6 +44,7 @@
         "privileged": true
     }
 }'
+TIMEOUTS='["commit-format": 10]'
```

Comment:
A good practice for variables that have measurement units is to write the unit in the name. This would remove the need to wonder what is it measured in, and makes the code easier to use. The timeout can be renamed to `TIMEOUTS_MIN`.

Reply:
Renamed it, thanks!